src/qibo/tests/test_hamiltonians.py

```diff
@@ -263,13 +266,13 @@ def test_tfim_model_hamiltonian(model, kwargs, filename, numpy):
 
 
 @pytest.mark.parametrize("nqubits", [3, 4])
-@pytest.mark.parametrize("model", [TFIM, XXZ, Y])
+@pytest.mark.parametrize("model", [TFIM, XXZ, Y, MaxCut])
 def test_trotter_hamiltonian_to_dense(nqubits, model):
     """Test that Trotter Hamiltonian dense form agrees with normal Hamiltonian."""
     local_ham = model(nqubits, trotter=True)
     target_ham = model(nqubits, numpy=True)
     final_ham = local_ham.dense
-    np.testing.assert_allclose(final_ham.matrix, target_ham.matrix)
+    np.testing.assert_allclose(final_ham.matrix, target_ham.matrix, atol=1e-15)
```

Comment:
Just a note that I was doing some checks yesterday and this tolerance may still fail if you increase `nqubits` for example to 7 or 10.